src/Integration.UnitTests/Persistence/SolutionBindingDataReaderTests.cs

```diff
@@ -57,19 +58,11 @@ public void TestInitialize()
         }
 
         [TestMethod]
-        public void Ctor_NullSerializer_Exception()
```

Comment:
Removed the "null arguments in constructor" tests as it is now MEF-created, and MEF will refuse to call the constructor with nulls (unless one of the arguments is marked as `ImportMany`).